build.bat

```diff
@@ -146,15 +146,15 @@ exit /b
 
 :build_data
 rem call :build_essay
+if not defined BASH_DIR (
+  set BASH_DIR=C:\Program Files\Git\usr\bin

```

Comment:
How about adding the path to `DEVTOOLS_PATH`?
The variable is already appended to `PATH` at the beginning of `build.bat`, and can be defined by user in `env.bat` for multiple dev tools. We don't need to set `PATH` twice or change `build.bat` every time we need something new.
Guessing the default path doesn't help when user do not have Git bash installed, or installed elsewhere.
In that case just let `cmd` complain `bash` not found.
Like for other tools, we can add its default install path to `env.bat.template` for convenience.

Reply:
This is not based on the modified `build.bat` in `build` branch, I can not add it here.
I will definitely do that when the new `build.bat` is done.